src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -96,8 +96,13 @@ function filter (port, secbindings, secrules, direction)
          end
       end
    end
-   if #rules > 0 then return { rules = rules }
-                 else return nil end
+   if type == "stateless" then
```

Comment:
@n-nikolaev No rules or no filter configs? E.g. the first case (an empty filter config) would drop all packets.